apps/qubit/modules/settings/actions/globalAction.class.php

```diff
@@ -213,6 +219,43 @@ protected function updateGlobalSettings()
       }
     }
 
+    // Identifier mask enabled
+    if (null !== $identifierMaskEnabled = $thisForm->getValue('identifier_mask_enabled'))
+    {
+      if (null === $setting = QubitSetting::getByName('identifier_mask_enabled'))
+      {
+        $setting = new QubitSetting;
```

Comment:
oh weird, I'm not sure why I did that hahaha. I was mostly just copying what other things were doing in that file. I'll remove it.